core/src/main/java/br/com/brainweb/interview/core/features/hero/HeroService.java

```diff
@@ -1,4 +1,47 @@
 package br.com.brainweb.interview.core.features.hero;
 
+import java.util.List;
+import java.util.UUID;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import br.com.brainweb.interview.core.features.powerstats.PowerStatsService;
+import br.com.brainweb.interview.model.Hero;
+import br.com.brainweb.interview.model.PowerStats;
+
+@Service
 public class HeroService {
```

Comment:
Faltou um item obrigatório :sob: 
O método que faz a comparação entre os heróis.

Reply:
Outro ponto importante é que as camadas compartilham sempre o mesmo objeto.

Ex: classe Hero é utilizada em todos os pontos do sistema do Controller até o Repository, isso não é uma boa prática pois podem ser necessárias mudanças nesse objeto que acabariam quebrando o contrato da API. (adicionar um novo atributo de negócio que não é necessário durante a criação do Hero por exemplo).